tests/acceptance/features/apiAuthApp/tokenUsingUserId.feature

```diff
@@ -111,3 +111,8 @@ Feature: create auth-app token using user-id
       """
       error parsing expiry. Use e.g. 30m or 72h
       """
+
+
+  Scenario: non-existent user tries to create auth-app token for another user using user-id
```

Comment:
move this with above non-existent user scenario or bring that one here. just grouping